flang/lib/Frontend/CompilerInvocation.cpp

```diff
@@ -1568,6 +1558,14 @@ bool CompilerInvocation::createFromArgs(
     success = false;
   }
 
+  // User-specified or default resource dir
+  if (const llvm::opt::Arg *a =
+          args.getLastArg(clang::options::OPT_resource_dir))
+    invoc.resourceDir = a->getValue();
+  else
+    invoc.resourceDir = clang::driver::Driver::GetResourcesPath(
```

Comment:
I've just started trying this patch, and I see a build failure suggesting that this should be `clang::GetResourcesPath`.

Reply:
There have been several reverts and re-applies of this patch: https://github.com/llvm/llvm-project/pull/169397. The pre-merge CI says it is fine: https://github.com/llvm/llvm-project/actions/runs/19701948097/job/56440119613?pr=169638 on the base patch as-is.

Reply:
Merged-in the latest main which includes the re-land (https://github.com/llvm/llvm-project/commit/d090311aa7df7cbd93412c31685893aefc4ad341). Should resole the issue unless reverted again.